hub/package-manager/winget/configure.md

```diff
@@ -19,7 +19,7 @@ The **configure** command of the [winget](./index.md) tool uses a [WinGet Config
 ## Prerequisites
 
 - Windows 10 RS5 or later, and Windows 11.
-- Winget version xxx or later?
+- Winget version v1.4.11071 or later.
```

Comment:
WinGet configuration was first introduced as an experimental feature in WinGet preview version [v1.5.1361-preview](https://github.com/microsoft/winget-cli/releases/tag/v1.5.1361-preview). So, the correct statement would be either 
`WinGet version v1.5.1361-preview or later` or if we want to include only stable versions then we can say `WinGet version v1.5.1572 or later.`

Reply:
Thank you for taking a look. Changed to:
v1.5.1572